core/color.js

```diff
@@ -0,0 +1,467 @@
+/**
+* @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+* For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+*/
+/**
+* @fileoverview Defines the {@link CKEDITOR.tools.style.Color} normalizer class
+* 		that parse color string code other formats.
+*/
+
+/**
+*Basic definitions
+*HEX 	 -> valid HEX Color Value with # value: #FFF or #FFFFFF
+*3-HEX -> HEX but with only 3 characters value: #FFF
+*6-HEX -> HEX but with exactly 6 characters value: #FFFFFF
+*/
```

Comment:
This comment shouldn't float around here like that. It could be moved to class definition/description or to relevant method docs.

Reply:
And the formatting is incorrect, it should be:

```
/**
 * Foo
 * Bar
 */
```

Reply:
Fixed comment formating. Finally, I moved it to the test file, because they are used in tests cases names.